src/main/webapp/src/app/find-gardens-page/find-gardens.component.html

```diff
@@ -0,0 +1,154 @@
+<owl-carousel-o [options]="customOptions">
+  <ng-template carouselSlide>
+    <mat-card class="mat-elevation-z8 cards">
+      <mat-card-header>
+        <mat-card-title class="title"><h2>Buffalo Garden</h2></mat-card-title>
+      </mat-card-header>
+
+      <img
+        class="garden-image"
+        src="https://images.unsplash.com/photo-1515150144380-bca9f1650ed9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+      />
+      <div class="avatar-image">
+        <img
+          mat-card-avatar
+          class="avatar"
+          src="https://images.unsplash.com/photo-1571442463800-1337d7af9d2f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+        />
+      </div>
+
+      <h3 class="name">Mary Brown</h3>
+
+      <mat-card-content>
+        <p>
+          Hello! This is my garden that I have been running for 6 years.
+          Currently in my garden I have...
+        </p>
+        <ul>
+          <li>Peas</li>
+          <li>Carrots</li>
+          <li>Potatoes</li>
+        </ul>
+        <p>
+          We would love to have more help around here!
+        </p>
+      </mat-card-content>
+      <mat-card-actions>
+        <button mat-button class="button">Join Garden</button>
```

Comment:
Food for thought: Does this button change if a user has already joined the garden? Or will garden cards not surface for gardens that a user is already a part of? Is there a separate place where a user might be able to leave a garden?

Reply:
Good question. So if the user clicks the join garden button then the garden will be added to the list of gardens that the user is a member of and will no longer be shown on the find gardens page. That page will specifically only show gardens in the user's area that they are not a member of. There is going to be a place where a user can leave the garden we just have to figure out specifically where that place would be but as a thought it should be an option when the user clicks on a garden in their my garden page. So the user will be able to see general information and be able to navigate to more specific info like the schedule and the plant data but at the end of the page there should be a red "Leave Garden" button.